static/sass/patterns/_code.scss

```diff
@@ -1,6 +1,360 @@
+// sass-lint:disable no-color-literals
+
 // Code Patterns
 // =============
 
-.highlight {
-  margin-bottom: 1rem;
+// See: http://pygments.org/docs/styles/
+// And: https://github.com/richleland/pygments-css
+.code {
+  background-color: color('background');
+  color: color('text');
+  margin-bottom: size('gutter');
+
+  // Generated by Pygments CSS Theme Builder
+  // https://jwarby.github.io/jekyll-pygments-themes/builder.html
+
+  // Punctuation
+  .p {
+    color: #7d8196;
```

Comment:
I'd vote for either marking these as temporary (if we want to come back to more detailed color decisions) or moving them into a map variable, something like `$highlight-colors`.